content/develop/connect/clients/client-side-caching.md

```diff
@@ -0,0 +1,187 @@
+    and [`ZRANDMEMBER`]({{< relref "/commands/zrandmember" >}}). By design, these commands
+    give different results each time they are called.
+-   Search and query commands (with the `FT.*` prefix), such as
+    [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search).
+
+You can use the [`MONITOR`]({{< relref "/commands/monitor" >}}) command to
+check the server's behavior when you are using CSC. Because `MONITOR` only
+reports activity from the server, you should find that the first cacheable
+access to a key causes a response from the server. However, subsequent
+accesses are satisfied by the cache, and so `MONITOR` should report no
+server activity if CSC is working correctly.
+
+## What data gets cached for a command?
+
+Broadly speaking, the data from the *specific response* to a command invocation
+gets cached after it is used for the first time. Subsets of that data
+or values calculated from it are retrieved from the server as usual and
+then cached separately. For example:
+
+-   The whole string retrieved by [`GET`]({{< relref "/commands/get" >}})
+    is added to the cache. Parts of the same string retrieved by
+    [`SUBSTR`]({{< relref "/commands/substr" >}}) are calculated on the
+    server the first time and then cached separately from the original
+    string.
+-   Using [`GETBIT`]({{< relref "/commands/getbit" >}}) or
+    [`BITFIELD`]({{< relref "/commands/bitfield" >}}) on a string
+    caches the returned values separately from the original string.
+-   For composite data types accessed by keys
+    ([hash]({{< relref "/develop/data-types/hashes" >}}),
+    [JSON]({{< relref "/develop/data-types/json" >}}),
+    [set]({{< relref "/develop/data-types/sets" >}}), and
+    [sorted set]({{< relref "/develop/data-types/sorted-sets" >}})),
+    the whole object is cached separately from the individual fields.
+    So the results of `HGETALL mykey` and `HGET mykey myfield` create
+    separate entries in the cache.
+-   Ranges from [lists]({{< relref "/develop/data-types/lists" >}}),
+    [streams]({{< relref "/develop/data-types/streams" >}}),
+    and [sorted sets]({{< relref "/develop/data-types/sorted-sets" >}})
+    are cached separately from the object they form a part of. Likewise,
+    subsets returned by [`SINTER`]({{< relref "/commands/sinter" >}}) and
+    [`SDIFF`]({{< relref "/commands/sdiff" >}}) create separate cache entries.
+-   For multi-key read commands such as [`MGET`]({{< relref "/commands/mget" >}}),
+    the ordering of the keys is significant. For example `MGET name:1 name:2` is
+    cached separately from `MGET name:2 name:1` because the server returns the
+    values in the order you specify.
+-   Boolean or numeric values calculated from data types (for example 
+    [`SISMEMBER`]({{< relref "/commands/sismember" >}})) and
+    [`LLEN`]({{< relref "/commands/llen" >}}) are cached separately from the
+    object they refer to.
+
+
+## Enabling CSC
```

Comment:
This is not required because the client library enables the tracking at the connection establishment time. So it is necessary to just configure the connection